index/ask/ask.go

```diff
@@ -201,12 +201,12 @@ func generateIndex(ctx context.Context, api *apistruct.FullNodeStruct) (*IndexSn
 			defer func() { <-rateLim }()
 			ictx, cancel := context.WithTimeout(ctx, qaTimeout)
 			defer cancel()
-			pid, err := api.StateMinerPeerID(ictx, addr, types.EmptyTSK)
```

Comment:
`StateMinerPeerID` was removed from Lotus API. 
Now it seems that you should make a more general API call for miner's info which has the PeerID.
The returned PeerID still seems to have some extra problems when used with QueryAsks (as mentioned in PR description about index-tests).